packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -40,9 +40,9 @@ export default function PostOptionsMenu({
   onRemovePost,
 }: PostOptionsMenuProps): ReactElement {
   useFeedSettings();
-  const { user } = useContext(AuthContext);
+  const { trackEvent } = useContext(AnalyticsContext);
   const { reportPost, hidePost } = useReportPost();
-  const { unfollowSource, blockTag } = useMutateFilters(user);
+  const { onUnfollowSource, onBlockTags } = useTagAndSource({ origin: 'feed' });
```

Comment:
Let's be more explicit here just in case

```suggestion
  const { onUnfollowSource, onBlockTags } = useTagAndSource({ origin: 'post context menu' });
```